src/component/playground/Playground/PlaygroundResultsTable/FeatureResultInfoPopoverCell/FeatureStrategyList/StrategyList/StrategyItem/StrategyExecution/helpers.ts

```diff
@@ -0,0 +1,11 @@
+export const getMappedParam = (key: string) => {
+    switch (key.toUpperCase()) {
+        case 'USERIDS':
+            return 'userId';
+        case 'IPS':
+            return 'remoteAddress';
+        case 'HOSTNAMES':
```

Comment:
There is no argument to return here: hostname can't be set via the context. Should we remove it completely? 🤔

Reply:
Removed